subprojects/core-api/src/main/java/org/gradle/api/flow/BuildWorkResult.java

```diff
@@ -25,11 +25,11 @@
  * Summary of the result of executing the
  * {@link StartParameter#getTaskRequests() requested tasks}.
```

Comment:
Should javadocs for this class also be reworded? They seem to be pretty heavily task-focused now.

Reply:
Yes, thanks, @mlopatkin.

Reply:
Done.